### PR TITLE
Add a step to the RMG about podcheck

### PR DESCRIPTION
**Pull request by khwilliamson**
It's quick to skim the podcheck summary of errors, and some things, like broken links should really be fixed before the release.

### CODE REVIEW

File: Porting/release_managers_guide.pod [hunk 1]
```diff
@@ -397,6 +397,21 @@ colon-delimited versions to use for comparison.  For example:
...
+release.  To see what it is ignoring, run
+
+ perl t/porting/podcheck.t --counts
+
+Any problems listed as pedantic aren't worth your time investigating.
+These have a C<?> at the beginning of the text, or are for the too-long
+verbatim lines.
+
+But other warnings should be.  In particular, a broken link can well
+mean that someone clicking on the pod in a web page will get a 404.
```
Comment (demerphq) [inline Porting/release_managers_guide.pod:414]: There should be a blank line after this one and before the next =head3

**Review by demerphq — CHANGES_REQUESTED** [review]
I don't entirely get the need for this.  As part of the release process we tell people to run test_porting, which runs these checks. 

I don't object to it either, I am just not sure why it is needed.

File: Porting/release_managers_guide.pod (hunk 1 above)
Comment (khwilliamson) [reply Porting/release_managers_guide.pod:414]: thanks

**Comment by khwilliamson** [comment]
The impetus for this was, IIRC, that 5.36.0 shipped with broken links.  As the text says, people wrongly silence podcheck warnings, and this adds a quick double check that that hasn't happened.

**Comment by khwilliamson** [comment]
I've added to the text more, but can't get github to accept it without a merge commit.  So I'm closing this request, and opening a replacement

**Pull request closed without merging**